### ISSUE TITLE
[SampleCard-FlightDetails.json; Config-Default]: Screen reader announcement is not descriptive enough for "SFO" and "JFK" and does not describe the topic or purpose of the element.

### ISSUE THREAD

**Issue by Bhaskar301101** [microsoft/AdaptiveCards#8477]
### Target Platforms

Android

### SDK Version

 2023.03.31.1

### Application Name

Adaptive Card

### Problem Description

Test Environment:
OS: Android 12
Device: S21+
Screen reader: Talkback
AdaptiveCard Android App version:  2023.03.31.1

Pre-requisite:
Import-Sample Cards->FlightDetails.json; Config-Default

Repro Steps:

1. Turn on the Talkback.
2. Navigate to the Visual.
3. Navigate to the sample card.
4. Using swipe gesture, swipe left to right in the card.
5. Navigate to "SFO" and "JFK" text element.
6. Observe the issue.

Actual Result:
Screen reader announcement is not descriptive enough for "SFO" and "JFK" and does not describe the topic or purpose of the element.
Screen reader only announces "SFO" and "JFK" when focus moves to the said text block elements.

Expected Result:
Screen reader announcement should describe the purpose of "SFO" and "JFK" text elements.
Forex, Screen reader should announce like as 'Where from San Francisco international airport (SFO)' and 'Where to John F Kennedy international airport (JFK)'.

Note:
This issue is also repro on Time and date text block, "Screen reader does not announce the label relationship of 'origin' and 'destination' names with respective 'time and date' details text block content."
PFA for reference 'Note-time and date.mp4'

User Experience:
The screen reader users will get impact and might get confuse when there is no heading or label info provided for text block element describing its purpose.

WCAG Reference:
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “,

### Screenshots

https://user-images.githubusercontent.com/89897257/232744072-0c40922e-572d-4fdf-ac33-e2e1503a16bf.mp4

https://user-images.githubusercontent.com/89897257/232744097-407ca3ab-0935-4c69-9a5a-8761ddd6ce74.mp4

### Card JSON

```JSON
NA
```

**Comment by VikrantSingh01** [comment]
@ardlank : Can you check this a11y issue.
